README.md

```diff
@@ -1,21 +1,38 @@
-# digitalstrategie
-
-[![Coverage Status](https://coveralls.io/repos/github/liqd/digitalstrategie/badge.svg?branch=main)](https://coveralls.io/github/liqd/digitalstrategie?branch=main)
-
-Welcome to digitalstrategie
+# Gemeinsam Digital: Berlin
 
 ## How to run
 
-Install dependencies
+#### Install dependencies
 ```
 $ make install
 ```
 
-### Start a local server:
+#### Start a local server:
+```
+$ make watch
+```
+
+Start local server with elastic search:
+
+Open two terminal windows.
+1. Start elastic search in the first one:
+```
+$ sudo docker run --name es01-test -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "logger.level=DEBUG" docker.elastic.co/elasticsearch/elasticsearch:7.17.10
```

Comment:
if we remove the `--name` argument we don't need the part with the deletion below

Reply:
ah, sorry, another thing which is probably nice: replace the `-p` wiith `-p 127.0.0.1:9200:9200 -p 127.0.0.1:9300:9300`